modules/ROOT/assets/attachments/sg-bootstrap.yaml

```diff
@@ -166,14 +201,38 @@ properties:
         type: object
         description: Group in which to specify any API HTTPS configuration properties
         properties:
-          allow_insecure_tls_connections:
-            type: boolean
-            default: false
+          # allow_insecure_tls_connections:
+          #   type: boolean
+          #   default: false
+          #   description: |+
+          #     Secure TLS connections are used by default for all REST API dialogs.
+
+          #     If necessary, for testing purposes, you can disable this by setting this property to ```true```.
+          #     *Note* tha insecure connections should NOT be used in production deployments.
+          tls_minimum_version:
+            type: string
+            default: 'tlsv1.3'
+            summary: The minimum allowable TLS version for the REST APIs
             description: |+
-              Secure TLS connections are used by default for all REST API dialogs.
+              Enforce a minimum TLS version to be used in replications with Couchbase Lite.
 
-              If necessary, for testing purposes, you can disable this by setting this property to ```true```.
-              *Note* tha insecure connections should NOT be used in production deployments.
+              Possible values are:
+
+              - "tlsv1"
+              - "tlsv1.1"
+              - "tlsv1.2"
+              - "tlsv1.3"
+
+          tls_cert_path:
+            type: string
+            summary: The TLS certificate file to use for the REST APIs
+            description: |+
+              Absolute or relative path on the filesystem to the TLS certificate file, if TLS is used to secure Sync Gateway connections, or "nil" for plaintext. A relative path is from the directory that contains the Sync Gateway executable file.
+          tls_key_path:
+            type: string
+            summary: The TLS key file to use for the REST APIs
+            description: |+
+              Absolute or relative path on the filesystem to the TLS private key file, if TLS is used to secure Sync Gateway connections, or "nil" for plaintext. A relative path is from the directory that contains the Sync Gateway executable file.
```

Comment:
The user would not use TLS with Sync Gateway by either not defining `tls_key_path` and `tls_cert_path` or by leaving the values empty (ie. `tls_key_path: ""` in the config file). By putting "nil" as the value, SGW will try to find a certificate called "nil". Putting nil as the value (ie. `tls_key_path: nil`) will error as that is invalid JSON.

It could also be mentioned here that the certificates are SSL but that would need to be confirmed by @rajagp .

Reply:
Replaced with omit instruction-- Done -- https://github.com/couchbase/docs-sync-gateway/pull/541